resources/default-settings/player.default.yaml

```diff
@@ -0,0 +1,28 @@
+# IMPORTANT:
+# Always use relative path from settingsfile `../../`, but do not use relative paths with `~/`.
+# Sole (!) exception is in playermpd.mpd_conf
+players:
+  content:
+    audiofile: /../../shared/audiofolders/audiofiles.yaml
+playermpd:
+  host: localhost
+  status_file: ../../shared/settings/music_player_status.json
+  second_swipe_action:
+    # Note: Does not follow the RPC alias convention (yet)
+    # Must be one of: 'toggle', 'play', 'skip', 'rewind', 'replay', 'none'
+    alias: toggle
+  library:
+    update_on_startup: true
+    check_user_rights: true
+  mpd_conf: ~/.config/mpd/mpd.conf
+playerspot:
+  host: localhost
+  status_file: ../../shared/settings/spotify_player_status.json
+  oauthcache:  ../../shared/spotify/.spotipyoauthcache
+  second_swipe_action:
```

Comment:
Is it a real use case that mpd and Spotify have a different second swipe behavior?